src/PolyhedralGeometry/iterators.jl

```diff
@@ -1,14 +1,16 @@
 ################################################################################
 ######## Scalar types
 ################################################################################
-const scalar_types = Union{fmpq, nf_elem}
+const scalar_types = Union{fmpq, nf_elem, Float64}
 
 const scalar_type_to_oscar = Dict{String, Type}([("Rational", fmpq),
-                                ("QuadraticExtension<Rational>", nf_elem)])
+                                ("QuadraticExtension<Rational>", nf_elem),
+                                ("Float", Float64)])
 
 const scalar_type_to_polymake = Dict{Type, Type}([(fmpq, Polymake.Rational),
                                     (nf_elem, Polymake.QuadraticExtension{Polymake.Rational}),
-                                    (Union{fmpq, nf_elem}, Polymake.QuadraticExtension{Polymake.Rational})])    # needed for Halfspace{nf_elem} etc
+                                    (Union{fmpq, nf_elem}, Polymake.QuadraticExtension{Polymake.Rational}),
+                                    (Float64, Float64)])    # needed for Halfspace{nf_elem} etc
```

Comment:
```suggestion
                                    (Union{fmpq, nf_elem}, Polymake.QuadraticExtension{Polymake.Rational}),    # needed for Halfspace{nf_elem} etc
                                    (Float64, Float64)])
```
to avoid some confusion